sdk/src/main/java/com/oursky/authgear/latte/fragment/LatteFragment.kt

```diff
@@ -42,19 +36,21 @@ internal abstract class LatteFragment<T>() : Fragment(), LatteHandle<T> {
     private val webContentsDebuggingEnabled: Boolean
         get() = requireArguments().getBoolean(KEY_WEBSITE_INSTPECTABLE)
 
-    internal constructor(id: String, url: Uri, redirectUri: String, webContentsDebuggingEnabled: Boolean) : this() {
+    internal constructor(context: Context, id: String, url: Uri, redirectUri: String, webContentsDebuggingEnabled: Boolean) : this() {
         arguments = Bundle().apply {
             putString(KEY_ID, id)
             putString(KEY_URL, url.toString())
             putString(KEY_REDIRECT_URI, redirectUri)
             putBoolean(KEY_WEBSITE_INSTPECTABLE, webContentsDebuggingEnabled)
         }
+        webView = WebView(context, WebViewRequest(url = url, redirectUri = redirectUri), webContentsDebuggingEnabled)
```

Comment:
This constructor would not be called if Android is re-creating the fragement (the 0-parameter constructor is used instead). Since fragment re-creation is not well supported in Latte at the moment, just FYI.

Reply:
Fragment re-creation would be handled in another PR (if we have time)